modules/ROOT/pages/define-parameters.adoc

```diff
@@ -455,6 +455,122 @@ As a summary, here’s a simple flow chart to aid with the decision process of h
 
 image::mule-sdk-dsl-parameter-pojos.png[Decision tree for POJOs and parameter groups]
 
+* Implementing *equals* and *hashCode* methods
+
+POJO parameters (regardless if they are used with the @ParameterGroup annotation) *must* override the *equals* and
```

Comment:
I would say that the classes are the ones that must override the method instead of the parameters and the parameter groups.